packages/stark-ui/src/modules/table/components/_table-theme.scss

```diff
@@ -0,0 +1,20 @@
+/* ============================================================================== */
+/*  S t a r k   A p p   T a b l e                                                  */
+/* ============================================================================== */
+/* stark-ui: src/modules/app-menu/components/_table-theme.scss */
+
+.stark-table {
+  tr {
+    &:hover td {
+      background-color: mat-color($grey-palette, 400);
+      color: mat-contrast($grey-palette, 400);
```

Comment:
Can we set a lighter gray on hover? And also.. can we leave the color of the text unchanged?
Basically as it was in the old Stark and I think this is also what the Material Design guideline suggests...